.gitignore

```diff
@@ -60,4 +60,7 @@ atlassian-ide-plugin.xml
 com_crashlytics_export_strings.xml
 crashlytics.properties
 crashlytics-build.properties
-fabric.properties
\ No newline at end of file
+fabric.propertiesa
```

Comment:
What is this `.propertiesa` entry? I appreciate you adding the VS Code settings to be excluded, but I don't understand this bit.

Reply:
it was a typo for sure. fixing it..